src/components/category/Category.js

```diff
@@ -1,10 +1,34 @@
 import React from 'react';
 import './category.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../../redux/categories/categories';
 
-const Categories = () => (
-  <div className="status-log">
-    <button type="button" id="updated-log" name="update">Check Status</button>
-  </div>
-);
+const Categories = () => {
+  const categories = useSelector((state) => state.reducer);
+
+  const dispatch = useDispatch();
+
+  return (
+    <div className="status-log">
+      <div>
+        {categories.map((category) => (
+          <p key={category}>
+            {category}
+          </p>
+        ))}
+      </div>
```

Comment:
- [ ] Great job on making the books section perfectly 👏 for the categories, we're supposed to see a button, when that button is clicked it the action creator `underConstruction` is dispatched and we see the state `Under Construction`. However, the app crashes and I get the following error 👇 

![image](https://user-images.githubusercontent.com/50721479/203832391-b8738837-42f3-4f39-8764-7fe98380394a.png)

hint: I think the state slice that you want to get here is `categoriesReducer` and not `reducer` (line 7). 
hint2: The state is a string and not an iterable so you can't use `map` on it 👍

Reply:
@ha-manel Thank you for the timely response and for providing hints. I have corrected the issues and everything seems to be running fine.

Reply:
awesome! good work 😁